Symfony/CS/Tests/IntegrationTest.php

```diff
@@ -35,4 +37,99 @@ protected static function getTempFile()
     {
         return self::getFixturesDir().DIRECTORY_SEPARATOR.'.tmp.php';
     }
+
+    /**
+     * @dataProvider provideBasicSources
+     */
+    public function testBasicFiles($source, array $whiteListFixerNames = array())
+    {
+        $this->doTest($source, $whiteListFixerNames);
+    }
+
+    public function provideBasicSources()
+    {
+        return array(
+            array('    '),
+            array("       <?php\n"),
+            array('<?php ', array('header_comment')),
+            array("<?php\n", array('header_comment')),
+            array("<?php\n\n\n", array('eof_ending', 'header_comment')),
+            array('<?php //', array('no_empty_comment', 'eof_ending', 'header_comment')),
+            array('<?php /**/  /**/', array('no_empty_comment', 'eof_ending', 'header_comment')),
+        );
+    }
+
+    /**
+     * @dataProvider provideBasicSources
+     */
+    public function testBasicFilesWithShortTag($source, array $whiteListFixerNames = array())
+    {
+        if (!ini_get('short_open_tag')) {
+            $this->markTestSkipped('PHP short open tags are not enabled.');
+
+            return;
+        }
+
+        $this->doTest($source, $whiteListFixerNames);
+    }
+
+    public function provideBasicSourcesWithShortTag()
+    {
+        return array(
+            array('<?= 1?>', array('short_echo_tag', 'php_closing_tag')),
+            array('<?=$a, $b, $c?>', array('short_echo_tag', 'php_closing_tag')),
+        );
+    }
+
+    /**
+     * @param string   $source
+     * @param string[] $whiteListFixerNames
+     *
+     * @dataProvider provideBasicSources
+     */
+    private function doTest($source, array $whiteListFixerNames = array())
```

Comment:
Please, could you provide some information for that method?
on 2.x we have it: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Test/AbstractIntegrationTestCase.php#L152

but I don't think they are combatible, am I right ?